stemcell_builder/stages/system_azure_wala/apply.sh

```diff
@@ -4,12 +4,13 @@ set -e
 
 base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
+source $base_dir/lib/prelude_bosh.bash
```

Comment:
Since this is only used for the `run_in_bosh_chroot` on line 41 (which is not needed, as far as i can tell), it should be removed.

Reply:
resolved.